res/css/views/auth/_AuthBody.scss

```diff
@@ -78,7 +78,8 @@ limitations under the License.
     margin-bottom: 10px;
 }
 
-.mx_AuthBody_fieldRow > * {
+.mx_AuthBody_fieldRow > *,
+.mx_AuthBody_fieldRow > .mx_Field {
```

Comment:
is `.mx_Field` not captured by `*`?

Reply:
The line `.mx_AuthBody_fieldRow > *` was enough for things that were not `mx_Field`, and indeed it still does also _match_ `mx_Field`. However, for the case of `mx_Field`, I actually need to override the default margin of `mx_Field` (which comes after this rule set), so I needed to add something of higher specificity here.

`*` doesn't count for specificity, so `.mx_AuthBody_fieldRow > *` is specificity 10, while `.mx_AuthBody_fieldRow > .mx_Field` is specificity 20, which is enough here.

Having said all this, by the end of this PR, we've actually removed all the non-`mx_Field` things from the `fieldRow`s, so I can remove the first line which should be less confusing.

Reply:
Fixed in https://github.com/matrix-org/matrix-react-sdk/pull/2749/commits/dde36459e6ac912201ef24b567616cb34ce1d90f